_impls/libassh.md

```diff
@@ -0,0 +1,118 @@
+---
+title: libassh
+homepage: http://libassh.org/
+source-repository: https://hg.savannah.nongnu.org/hgweb/libassh/
+license: "[LGPL 2.1](https://git.libssh.org/projects/libssh.git/tree/COPYING)"
+first-release:
+    date: 2020-09-28
+latest-release:
+    version: 1.0
+    date: 2021-01-03
+#changelog: TODO
+client: yes
+server: yes
+library: both
+protocols:
+    cipher:
+	- 3des-cbc

```

Comment:
Could you please change the tabs to spaces?

Reply:
I changed to spaces in 6dfde26.